doc/api/plural/plural-generator.md

```diff
@@ -37,6 +50,18 @@ plural( 1 );
 
 plural( 2 );
 // > "other"
+
+// Ordinals
+plural = Globalize.pluralGenerator( { type. "ordinal" } );
```

Comment:
The coding style you used is correct according to http://contribute.jquery.org/style-guide/js/. But, can you change `( { type: "ordinal" } )` to `({ type: "ordinal" })` please? The extra space is a new rule and it's not in use by this repo yet.